feature: RP intermediary close to EUDIW from fork (#1140)
italia/eidas-it-wallet-docs: docs/en/onboarding-high-level.rst

 - **Certificate Issuance**: Issuance of certificates that authenticate entities, reference the registry for entitlement verification, support certificate transparency, and describe registration status for Relying Parties and Credential Issuers.
 - **Registry Publication**: Publication of all registered entities in the registry, with an online API (see :ref:`trust-infrastructure:Trust Infrastructure and Registry Integration`) that can be used to verify entity registration, and requested attributes.

Onboarding of Relying Party Intermediaries
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Relying Party Intermediaries undergo a dedicated onboarding process that grants them both the status of recognized Intermediary and the authority to onboard affiliated Relying Parties on behalf of the Trust Anchor. In the implementation profile described by these technical specifications, a Relying Party Intermediary is also an OpenID Federation Intermediate Entity (see the definition of **IT-Wallet Intermediary**). The process includes:

Reviewer: ```suggestion
Relying Party Intermediaries use a dedicated onboarding process that grants the status of recognized Intermediary and the authority to onboard subordinate Relying Parties. In the implementation profile described by these technical specifications, a Relying Party Intermediary matches an OpenID Federation Intermediate Entity (see the definition of **IT-Wallet Intermediary**). The process includes:
```

- **Administrative registration**: The Intermediary submits organizational and legal documentation demonstrating its eligibility to act on behalf of Relying Parties pursuant to Art. 5b(8) of the eIDAS2 Regulation (`EU_2024_1183`_).

> - **Technical registration**: The Intermediary registers its federation keys and endpoints, enabling it to publish its own Entity Configuration and issue Subordinate Statements for affiliated Leaves.

Reviewer: ```suggestion
- **Technical registration**: The Intermediary registers its federation keys and endpoints, enabling it to publish its own Entity Configuration and issue Subordinate Statements for subordinate Leaves.
```

> - **Issuance of the Intermediary Trust Mark**: Upon completion of the onboarding process, the Trust Anchor issues to the Intermediary a Trust Mark with identifier ``https://<federation_authority_domain>/trust_marks/federation-entity/rp_intermediary``. This Trust Mark is the verifiable evidence of the Intermediary's recognized role in the federation.

Reviewer: ```suggestion
- **Issuance of the Intermediary Trust Mark**: Upon completion of the onboarding process, the Trust Anchor issues to the Intermediary a Trust Mark with identifier ``https://<federation_authority_domain>/trust_marks/federation-entity/openid_credential_verifier_intermediary``. This Trust Mark is the verifiable evidence of the Intermediary's recognized role in the federation.
```

- **Authorization to issue Trust Marks**: The Trust Anchor updates its own Entity Configuration by including the Intermediary in the ``trust_mark_issuers`` attribute for the ``https://<federation_authority_domain>/trust_marks/federation-entity/openid_credential_verifier`` identifier, authorizing the Intermediary to issue Trust Marks to its affiliated Relying Parties.

> Following this onboarding, Relying Parties affiliated with the Intermediary complete their own onboarding through the Intermediary, which acts as an intermediate Registration Authority. The affiliated RP's Entity Configuration MUST include the Trust Mark issued by the Intermediary and MUST set ``authority_hints`` to the Intermediary's URL identifier.

Reviewer: ```suggestion
The affiliated RP's Entity Configuration MUST include the Trust Mark issued by the Intermediary and MUST set ``authority_hints`` using the Intermediary's URL identifier.
```
